src/lib/app.js

```diff
@@ -50,7 +50,6 @@ class App {
     koaApp.use(compress())
 
     const websocketRouter = this._makeWebsocketRouter()
-    koaApp.ws.use(logger)
```

Comment:
Is there a way to make it log activity in another way? Or do we just not want it to log activity?